aleph0/lib/file.ts

```diff
@@ -0,0 +1,49 @@
+import fs from "fs";
+import path from "path";
+import { consola } from "consola";
+
+export function readFile(filePath: string) {
+  return fs.readFileSync(filePath, "utf-8");
+}
+
+// from: https://github.com/nicoalbanese/kirimase/blob/master/src/utils.ts
+
+export function createFile(filePath: string, content: string) {
+  const resolvedPath = path.resolve(filePath);
+  const dirName = path.dirname(resolvedPath);
+
+  // Check if the directory exists
+  if (!fs.existsSync(dirName)) {
+    // If not, create the directory and any nested directories that might be needed
+    fs.mkdirSync(dirName, { recursive: true });
+    // consola.success(`Directory ${dirName} created.`);
```

Comment:
Let's remove unnecessary comments?

Reply:
sure. i just copy and pasted this from https://github.com/nicoalbanese/kirimase/blob/master/src/utils.ts